### PR TITLE
[WebProfilerBundle] Fix theme settings

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #28932 #28935
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by ro0NL** [comment]
Responsive:

image

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <style>
 #open-settings {
-    color: var(--text-muted);
```
Comment (ro0NL) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig]: udefined var

**Comment by ro0NL** [comment]
Seems if fixes #28935 as well :)

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig [hunk 2]
```diff
@@ -69,7 +69,7 @@
     padding: 0 0 5px;
 }
 .modal-content input, .modal-content .form-help {
-    margin-left: 15px;
+    margin-left: 15px;1
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig]: that looks wrong.

Comment (ro0NL) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig]: fixed

**Review by javiereguiluz — APPROVED** [review]
I've just tested this in a 4.2-beta1 app and it worked perfectly! Thanks a lot Roland.

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @ro0NL.

**Pull request merged**